openapi.yaml

```diff
@@ -8242,6 +8242,14 @@ components:
           description: Issues must be created before this date.
           format: date
           type: string
+        groups:
+          description: |
+            Calculate metrics separately for each JIRA Filter in the list.
+            Not compatible with other jira filters in the request:
```

Comment:
Note that this limitation is not expressed by the schema: our typescript generator would explode with not/anyOf or allOf/not combinations. The API will check this condition manually.